Language/Functions/Time/millis.adoc

```diff
@@ -66,6 +66,8 @@ void loop() {
 === Notes and Warnings
 Please note that the return value for millis() is of type `unsigned long`, logic errors may occur if a programmer tries to do arithmetic with smaller data types such as `int`. Even signed `long` may encounter errors as its maximum value is half that of its unsigned counterpart.
 
+On some architectures reconfiguration of timers may result in inaccurate millis() readings. AVR-based architectures program a timer to implement millis(): ArduinoCore-AVR programs TIM0 to generate an interrupt every 16384 clock cycles (clock divider set to 64, overflow every 256 ticks). The same holds for ArduinoCore-mega-AVR, except that it uses TIMx if MILLIS_USER_TIMx (0<=x<=3) is defined. The ArduinoCore-SAM(D) architectures program the Systick timer to generate an interrupt every 1 ms.
```

Comment:
```suggestion
Reconfiguration of the microcontroller's timers may result in inaccurate `millis()` readings. The "Arduino AVR Boards" and "Arduino megaAVR Boards" cores use Timer0 to generate `millis()`. The "Arduino ARM (32-bits) Boards" and "Arduino SAMD (32-bits ARM Cortex-M0+) Boards" cores use the SysTick timer.
```

- Reduce unnecessarily technical language to make information more approachable to the target reader
- Use standard formatting for inline code: https://raw.githubusercontent.com/arduino/reference-en/master/AsciiDoc_sample/AsciiDoc_Dictionary/AsciiDoc_Template-Dictionary.adoc
  (yes, I know this is not done consistently in the existing content, but we have been trying to improve on that over the recent years)
- Use human friendly boards platform names
- Use "Timer0" and "SysTick" timer names, which seem to be the official way to refer to them in prose:
  - https://onlinedocs.microchip.com/pr/GUID-93DE33AC-A8E1-4DD9-BDA3-C76C7CB80969-en-US-2/index.html?GUID-0BED8014-6B01-47DD-A495-68F9F70287A3
  - https://microchipdeveloper.com/32arm:samd21-systick